frontend/src/components/phones/dashboard/Dashboard.tsx

```diff
@@ -1,12 +1,179 @@
+/* eslint-disable @typescript-eslint/no-non-null-asserted-optional-chain */
 import { useRelayNumber } from "../../../hooks/api/relayNumber";
+import styles from "./PhoneDashboard.module.scss";
+import { CopyIcon, ForwardIcon, BlockIcon } from "../../../components/Icons";
+import { MouseEventHandler, useState } from "react";
 
 export const PhoneDashboard = () => {
   const relayNumberData = useRelayNumber();
+  const [justCopiedApiKey, setJustCopiedPhoneNumber] = useState(false);
+
+  const [enableForwarding, setEnableForwarding] = useState(
+    relayNumberData.data?.[0].enabled
+  );
+
+  const arrayPhoneNumber = relayNumberData.data?.[0].number.split("");
+  arrayPhoneNumber?.splice(2, 0, " (");
+  arrayPhoneNumber?.splice(6, 0, ") ");
+
+  const phoneNumberBracketed = arrayPhoneNumber?.join("");
```

Comment:
suggestion (non-blocking): I bet we're going to need to do this a *LOT* where we display the back-end E.164-formatted number in a more pretty display. Should we move this to a helper function like `e164ToDisplayNumber`?

note: I think I can also upate API endpoints to include something like a `friendly_number` field that formats an e.164 number into the display format most-applicable for the number's country.

Reply:
Yeah, moving it to a function makes sense.